components/renku_data_services/k8s/quota.py

```diff
@@ -30,11 +41,12 @@ def _quota_from_manifest(self, manifest: client.V1ResourceQuota) -> models.Quota
                 gpu_kind = igpu_kind
                 break
         memory_raw = manifest.spec.hard.get("requests.memory")
-        if memory_raw[-1] == "i":
+        # NOTE: Pydantic will fail to parse "Gi" but will be happy with "Gib", but Kubernetes works with Gi or G
```

Comment:
Would it not be safer to use the `parse_quantity()` util from the `kubernetes` library?

Reply:
I did not know about this. That is probably better.

Reply:
Technically, all fields from `manifest.spec.hard` are quantities, so `cpu` may also be specified as `2500m` (2.5).

Reply:
Done.